pkg/cmd/factory/default.go

```diff
@@ -107,6 +108,11 @@ func browser(f *cmdutil.Factory) cmdutil.Browser {
 	return cmdutil.NewBrowser(browserLauncher(f), io.Out, io.ErrOut)
 }
 
+func prompter(f *cmdutil.Factory) cmdutil.Prompter {
+	io := f.IOStreams
+	return cmdutil.NewPrompter(io.In, io.Out, io.ErrOut)
```

Comment:
On Windows, `io.Out` will be a wrapped `os.Stdout` in a Colorable stream, translating ANSI escape sequences to Windows equivalents for compatibility with Windows Console Host. Survey independently [does the same](https://github.com/AlecAivazis/survey/blob/master/terminal/output_windows.go). I'm not sure what will happen if you pass a Colorable out stream to Survey— will it additionally wrap it with its own colorable stream and somehow conflict? Or it might Just Work, but I advise that you apply care while testing.

Reply:
ah, i'll test on windows today.